<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VMECplot</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="Filename_label">
          <property name="text">
           <string>File Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FileName"/>
        </item>
        <item>
         <widget class="QLabel" name="PlotName_label">
          <property name="text">
           <string>Plot Quantity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="PlotList"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="PlotButtons">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>122</width>
              <height>200</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>Plot Options</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <widget class="QRadioButton" name="rho_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>R</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="pol_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Phi</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="tor_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="flux_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>100</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Phi-Z</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="poltor_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>120</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>R-Z</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="RZ_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>140</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>R-phi</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="ThreeD_button">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>170</y>
               <width>100</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>3D</string>
             </property>
            </widget>
            <widget class="QPushButton" name="savebutton">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>500</y>
               <width>113</width>
               <height>32</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Supported formats: eps, pdf, pgf, png, ps, raw, rgba, svg, svgz</string>
             </property>
             <property name="text">
              <string>Save Image</string>
             </property>
            </widget>
            <widget class="QTextEdit" name="saveas_filename">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>460</y>
               <width>104</width>
               <height>31</height>
              </rect>
             </property>
             <property name="toolTip">
              <string extracomment="Supported formats: eps, pdf, pgf, png, ps, raw, rgba, svg, svgz">Supported formats: eps, pdf, pgf, png, ps, raw, rgba, svg, svgz</string>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;(png)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QSlider" name="rhoslider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>R</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QSlider" name="uslider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Phi</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QSlider" name="vslider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="plot_widget">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
